CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+<p align="center">
+  <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+  </a>
+</p>
+
+# Contributing to the Sentry SDK for PHP
+
+We welcome contributions to sentry-php by the community. See the [Contributing to Docs](https://docs.sentry.io/contributing/) page if you want to fix or update the documentation on the website.
+
+## How to report a problem
+
+Please search the [issue tracker](https://github.com/getsentry/sentry-php/issues) before creating a new issue (a problem or an improvement request). Please also ask in our [Sentry Community on Discord](https://discord.com/invite/Ww9hbqr) before submitting a new issue. There is a ton of great people in our Discord community ready to help you!
+
+If you feel that you can fix or implement it yourself, please read a few paragraphs below to learn how to submit your changes.
+
+## Submitting changes
+
+- Setup the development environment.
+- Clone the sentry-php repository and prepare necessary changes.
+- Add tests for your changes into `tests/`.
+- Submit a pull request, targeting the `develop` branch if you added any new features. For bug fixes of the current release, please target the `master` branch instead.
+- Make sure to update the `CHANGELOG.md` file below the `Unreleased` heading.
+
+We will review your pull request as soon as possible.
+Thank you for contributing!
+
+## Development environment
+
+### Clone the repository
+
+```
+$ git clone git@github.com:getsentry/sentry-php.git
+```
+
+Make sure that you have PHP 7.2 installed, as this is the lowest version the SDK should work with.
+
+On macOS, we recommend using [Homebrew](https://brew.sh/) to install PHP. For Windows, we recommend ???.
```

Comment:
How do people install PHP on Windows these days? Is XAMPP still a thing?

Reply:
I think that either they use a tool like XAMPP or they download the pre-built PHP from https://windows.php.net or (most probably) they use Docker